deployments/helm/k8s-dra-driver/templates/kubeletplugin.yaml

```diff
@@ -76,6 +76,8 @@ spec:
         resources:
           {{- toYaml .Values.kubeletPlugin.containers.plugin.resources | nindent 10 }}
         env:
+        - name: NVIDIA_CTK_PATH
+          value: "{{ .Values.nvidiaCtkPath }}"
```

Comment:
Does this also require us to set a default in `values.yaml`?

Reply:
Also, is there a reason that this should be set at the root level of the `values.yaml` rather than specific to the kubelet plugin (same goes for `.Values.nvidiaDriverRoot` for that matter)?

Reply:
On:
> Does this also require us to set a default in `values.yaml`?
The `nvcdi` package will default to `/usr/bin/nvidia-ctk` if this is empty, but I would assume that an explicit entry in `values.yaml` would be better.

> Also, is there a reason that this should be set at the root level of the `values.yaml` rather than specific to the kubelet plugin (same goes for `.Values.nvidiaDriverRoot` for that matter)?

I think this depends on the API that the values file represents. I suppose from my perspective, the current options in the `kubeletplugin` section seem related to that pod specifically, whereas the driver root and this path are not specific to the plugin. Yes, they're *currently* only used there, but if another component is added here that requires these values where would be specify them then?

Reply:
> Does this also require us to set a default in `values.yaml`?

Good point. Fixed.

Reply:
> Also, is there a reason that this should be set at the root level of the `values.yaml` rather than specific to the kubelet plugin (same goes for `.Values.nvidiaDriverRoot` for that matter)?

Hmm, I can fix that. Where do they should go, in your opinion?

Reply:
@empovit I think this is probably more of a question for us internally than a serious issue with the PR. 

@klueska do you want to block this PR on defining some other top-level structure to define these kinds of properties (`driver`, `host`?) or are you ok with leaving this as a top-level value for the time being?